tests/foreman/cli/test_contentview.py

```diff
@@ -647,6 +647,55 @@ def test_positive_create_composite(self):
 
     @tier2
     @run_only_on('sat')
+    def test_positive_remove_version_by_id_from_composite(self):
+        """Create a composite content view and remove its content version by id
+
+        @id: 0ff675d0-45d6-4f15-9e84-3b5ce98ce7de
+
+        @assert: Composite content view info output does not contain any values
+
+        @CaseLevel: Integration
+        """
+        # Create new organization, product and repository
+        new_org = make_org({u'name': gen_alphanumeric()})
```

Comment:
1) Do we really need new organization and product? Can't we re-use them from setup section?
2) Do we need to have product and repository for these tests?